shared/video-sdk/authentication-workflow/project-implementation/cpp.mdx

```diff
@@ -1,34 +1,51 @@
+1. **Enable the user to specify a channel**
```

Comment:
Sorry, I am not using the test case where a user can edit the channel name and on channel update it should go again to fetch new token. Just to see the channel name in UI, is of no use.. That's why I have taken hard coded  channelName in startup only. No need of this Edit box. It doesn't serve any purpose here. Our main intension is to show the token generation in case of token is about to expire. Actually the case where user can edit the channel and token expired and go for fetching the new token with changed channel name  , is not our case..
Please note  We have not to make any condition where token expired actually , we are only handling when token is about to expire (). So even if I put the edit box and made the changes. The running token will not expired on changing the channel name. It will call the fetchToken() when it would be about to expire as per the value of `expireTime` . Yes when it would go for fetching the token before re-joining , then only it will fetch the new token with the new channel name. But it will not serve any use case here. @billy-the-fish  please suggest on this.